core/file_walker.go

```diff
@@ -60,9 +60,12 @@ func GetSubTree(path string, parent *File, readDir ReadDir, ignoredFolders map[s
 			files = append(files, file)
 		}
 	}
-	ret.Name = name
 	if parent != nil {
+		ret.Name = name
 		ret.Parent = parent
+	} else {
+		// Root dir
+		ret.Name = dirName + name
```

Comment:
Instead of setting the "root" dir name to e.g. `/home/leon` (when running `godu ~`), we could create a fake file: `home` which links to `leon`